test/test_api.py

```diff
@@ -542,14 +542,15 @@ def test28_create_upload_preset(self, mocker):
         """ should allow creating upload_presets """
         mocker.return_value = MOCK_RESPONSE
 
-        api.create_upload_preset(name=API_TEST_PRESET, folder="folder")
+        api.create_upload_preset(name=API_TEST_PRESET, folder="folder", live=True)
```

Comment:
@const-cloudinary , what will happen if I send "False"?

Reply:
the value live=0 will be sent to the server.